addons/mail/static/src/models/chat_window.js

```diff
@@ -135,6 +135,22 @@ registerModel({
             ev.stopPropagation();
             this.expand();
         },
+        /**
+         * Called when clicking on header of chat window. Usually folds the chat
+         * window.
+         */
+        onClickHeader(ev) {
+            if (!this.exists() || this.messaging.device.isMobile) {
```

Comment:
Rather than having this condition, perhaps you could simply remove the handler on mobile?

Reply:
Note that in Owl 1 it was possible to "remove" a handler by giving it `undefined`, but it's not even possible in Owl 2. The only way to conditionally have a handler would be to duplicate the node and add conditions one with the handler and one without (not great).